src/mqtt.client.ts

```diff
@@ -448,6 +453,15 @@ export class MqttClient<
             .catch(e => this.emitWarning(e));
     }
 
+    protected onPingReq() {
+        this.startFlow(incomingPingFlow() as PacketFlowFunc<ReadMap, WriteMap, any>)
+            .then(() => this.pingDebug(`PingPong @ ${Date.now()}`))
```

Comment:
This should say something different. Something like `Server-PingPong` as it's not the traditional pingpong.